lib/fs.js

```diff
@@ -2741,6 +2752,67 @@ function mkdtempSync(prefix, options) {
   return result;
 }
 
+/**
+ * Asynchronously copies `src` to `dest`. `src` can be a file, directory, or
+ * symlink. The contents of directories will be copied recursively.
+ * @param {string | Buffer | URL} src
+ * @param {string | Buffer | URL} dest
+ * @param {Object} [options]
+ * @param {() => any} callback
+ * @returns {void}
+ */
+function copy(src, dest, options, callback) {
+  if (typeof options === 'function' && !callback) {
+    callback = options;
+    options = {};
+  } else if (typeof options === 'function') {
+    options = { filter: options };
+  }
+  callback = makeCallback(callback);
+  options = options || {};
+  options = validateCopyOptions(options);
+  if (isBuffer(src)) {
+    src = src.toString();
+  }
+  if (isBuffer(dest)) {
+    dest = dest.toString();
+  }
+  src = getValidatedPath(src);
+  dest = getValidatedPath(dest);
+  src = pathModule._makeLong(src);
+  dest = pathModule._makeLong(dest);
```

Comment:
https://github.com/nodejs/node/blob/89f592cc82cd5c71d0e2389b75c1d36bbcd83c51/lib/path.js#L1539-L1541
```suggestion
  src = pathModule.toNamespacedPath(src);
  dest = pathModule.toNamespacedPath(dest);
```